reconcile/ldap_groups/integration.py

```diff
@@ -129,14 +149,32 @@ def get_integration_settings(query_func: Callable) -> LdapGroupsSettingsV1:
     def get_roles(self, query_func: Callable) -> list[RoleV1]:
         data = roles_query(query_func, variables={})
         roles = [role for role in data.roles or [] if role.ldap_group]
-        duplicates = find_duplicates(role.ldap_group for role in roles)
-        if duplicates:
+        if duplicates := find_duplicates(role.ldap_group for role in roles):
             for dup in duplicates:
                 logging.error(f"{dup} is already in use by another role.")
             raise ValueError("Duplicate ldapGroup value(s) found.")
         return roles
 
-    def fetch_desired_state(
+    def get_aws_groups(self, query_func: Callable) -> list[AWSGroupV1]:
+        data = aws_groups_query(query_func, variables={})
+        aws_groups = [
+            group
+            for group in data.aws_groups or []
+            if group.roles
+            and (group.account.sso is None or group.account.sso)
```

Comment:
`sso` default to True if not specified? Seems a dangerous default.

Reply:
You're right! Let me change it!